spec/acceptance/foreman_cli_plugins_spec.rb

```diff
@@ -3,41 +3,75 @@
-    }
-
-    if $facts['os']['family'] == 'RedHat' {
-      include foreman::cli::ansible
-      include foreman::cli::azure
-    }
-    include foreman::cli::discovery
-    include foreman::cli::remote_execution
-    include foreman::cli::tasks
-    include foreman::cli::templates
-    PUPPET
+  context 'for standard plugins' do
+    let(:pp) do
+      <<-PUPPET
+      class { 'foreman::cli':
+        foreman_url => 'https://foreman.example.com',
+        username    => 'admin',
+        password    => 'changeme',
+      }
+
+      if $facts['os']['family'] == 'RedHat' {
+        include foreman::cli::ansible
+        include foreman::cli::azure
+      }
+      include foreman::cli::discovery
+      include foreman::cli::remote_execution
+      include foreman::cli::tasks
+      include foreman::cli::templates
+      PUPPET
+    end
+
+    it_behaves_like 'a idempotent resource'
+
+    it_behaves_like 'hammer'
+
+    ['discovery', 'remote_execution', 'tasks', 'templates', 'katello'].each do |plugin|
+      package_name = case fact('os.family')
+                     when 'RedHat'
+                       fact('os.release.major') == '7' ? "tfm-rubygem-hammer_cli_foreman_#{plugin}" : "rubygem-hammer_cli_foreman_#{plugin}"
+                     when 'Debian'
+                       "ruby-hammer-cli-foreman-#{plugin.tr('_', '-')}"
+                     else
+                       plugin
+                     end
+
+      describe package(package_name) do
+        it { is_expected.to be_installed }
+      end
+    end
   end
 
-  it_behaves_like 'a idempotent resource'
+  if fact('os.family') == 'RedHat'
+    context 'for katello' do
+      let(:pp) do
+        <<-PUPPET
+        yumrepo { 'katello':
+          baseurl  => "http://yum.theforeman.org/katello/nightly/katello/el${facts['os']['release']['major']}/x86_64/",
+          gpgcheck => 0,
+        }
+
+        class { 'foreman::cli':
+          foreman_url => 'https://foreman.example.com',
+          username    => 'admin',
+          password    => 'changeme',
+        }
+
+        include foreman::cli::katello
+
+        Yumrepo['katello'] -> Class['foreman::cli::katello']
+        PUPPET
+      end
 
-  it_behaves_like 'hammer'
+      it_behaves_like 'a idempotent resource'
 
-  ['discovery', 'remote_execution', 'tasks', 'templates'].each do |plugin|
-    package_name = case fact('os.family')
-                   when 'RedHat'
-                     fact('os.release.major') == '7' ? "tfm-rubygem-hammer_cli_foreman_#{plugin}" : "rubygem-hammer_cli_foreman_#{plugin}"
-                   when 'Debian'
-                     "ruby-hammer-cli-foreman-#{plugin.tr('_', '-')}"
-                   else
-                     plugin
-                   end
+      it_behaves_like 'hammer'
 
-    describe package(package_name) do
-      it { is_expected.to be_installed }
+      package_name = fact('os.release.major') == '7' ? "tfm-rubygem-hammer_cli_katello" : "rubygem-hammer_cli_katello"
```

Comment:
```suggestion
      package_name = (fact('os.release.major') == '7' ? "tfm-" : "") + "rubygem-hammer_cli_katello"
```

Or

```suggestion
      package_name = "#{fact('os.release.major') == '7' ? "tfm-" : ""}rubygem-hammer_cli_katello"
```

Or, since it's already used elsewhere, define a regular variable that can also be used in the regular plugin test:
```ruby
scl_prefix = fact('os.release.major') == '7' ? "tfm-" : ""
```